crates/rspack_core/src/resolver/resolver_impl.rs

```diff
@@ -68,12 +83,16 @@ impl<'a> ResolveInnerOptions<'a> {
 #[derive(Debug)]
 pub enum Resolver {
   NodejsResolver(nodejs_resolver::Resolver, Arc<nodejs_resolver::Cache>),
-  // OxcResolver(oxc_resolver::Resolver),
+  OxcResolver(oxc_resolver::Resolver),
 }
 
 impl Resolver {
   pub fn new(options: Resolve) -> Self {
-    Self::new_nodejs_resolver(options)
+    if std::env::var("OXC_RESOLVER").is_ok() {
```

Comment:
use `experiments.rspackFuture.enableNewResolver` to control, so we can run  tests against new resolver and old resolver

Reply:
@hardfist  I thought we are omitting the `enable` part as we discussed before.

Reply:
sorry forget this name convention